BUILD.bazel

```diff
@@ -149,3 +150,22 @@ genrule(
     outs = ["pocket-ic"],
     cmd = "gunzip -c $< > $@",
 )
+
+# Upload artifacts to the CDN
+upload_artifacts(
+    name = "upload-artifacts",
+    testonly = True,
```

Comment:
something, somewhere, is marked as `testonly` which causes all those deps to be `testonly`. I'll look into this separately